compiler/rustc_incremental/src/persist/file_format.rs

```diff
@@ -28,12 +28,12 @@ const HEADER_FORMAT_VERSION: u16 = 0;
 /// the Git commit hash.
 const RUSTC_VERSION: Option<&str> = option_env!("CFG_VERSION");
 
-pub fn write_file_header(stream: &mut Encoder) {
+pub fn write_file_header(stream: &mut Encoder, sess: &Session) {
```

Comment:
```suggestion
pub fn write_file_header(stream: &mut Encoder, nightly_build: bool) {
```
Nit: not sure why you sometimes pass a session and sometimes a boolean.

Reply:
I forget exactly where, but some part of `rustc_incremental` spawns a thread before calling this and `Session` is not `Send` -> `&Session` is not `Sync`.

Reply:
I think I misread this the first time - do you want me to change both to booleans instead? That would work fine.

Reply:
>  do you want me to change both to booleans instead?

Done.